cardano-db-sync-extended/app/cardano-db-sync-extended.hs

```diff
@@ -29,6 +31,7 @@ main = do
   case cmd of
     CmdVersion -> runVersionCommand
     CmdRun params -> runDbSyncNode extendedDbSyncNodePlugin params
+    CmdValidate params -> runValidator params
```

Comment:
Why is this being added here? Is it not possible to do this validation in the `cardano-db-tool` program?

Reply:
The new validations added here need to import `Cardano.DbSync.LedgerState` which is found at `cardano-db-sync` package. This would create circular package dependencies, since `cardano-db-sync` already import `cardano-db`

I don't understand the last question: This is what this new validator does: it reads a state from a file and validate that against the db.

Reply:
I can totally see the point of moving `cardano-db-tool` from the `cardano-db` to the `cardano-db-sync` package (as a separate PR).